src/components/filters/input.tsx

```diff
@@ -14,7 +14,7 @@ interface Props {
   min?: number
   max?: number
   position?: "left" | "right"
-  apply: (filtersToApply) => void
+  apply: (event) => void
```

Comment:
I'm a bit late to the party but is there no way around using raw event here?
Other filters use the object interface and it would be good to have consistency...
Also, keep in mind that changes like this are breaking changes

Reply:
also late to the party. this was a breaking change :)